frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -54,18 +54,63 @@ const props = defineProps({
         type: Array as PropType<FamAccessControlPrivilegeGetResponse[]>,
         default: [],
     },
+    newUserInTable: {
+        type: String,
+        default: '',
+    },
 });
 
+type UserRoleOrAppAdminOrAccessControlPrivilege =
+    | FamApplicationUserRoleAssignmentGet
+    | FamAppAdminGetResponse
+    | FamAccessControlPrivilegeGetResponse;
+
+type UserRoleOrAppAdminOrAccessControlPrivilegeWithIsNew =
+    UserRoleOrAppAdminOrAccessControlPrivilege & {
+        isNewUser: boolean;
+    };
```

Comment:
Sorry, I am confused, what are these two **type**s and `isNewUser` for?
Also the name, do you mean `isNewUserAccess`?

Reply:
I removed then, as they are not needed anymore.